CMSgov/mint-app
# [EASI-3682] - Payment reconciliation frequency

# EASI-3682

Because this PR pulls into the git history of another ticket, use [this link](https://github.com/CMSgov/mint-app/compare/5bd0677..50720f6?expand=1) to view the real diff

## Changes and Description

[Figma](https://www.figma.com/file/M72DReE73SSXMN48WZfaKN/MINT---New-Model-Process-(Model-Intake)?type=design&node-id=7151-112511&mode=design&t=1ejww7jxmWrODCg9-0)

BE Changes
- Added new field to schema - `paymentReconciliationFrequency`
- Added `Continually`, `Other`, and `Note` string fields
- Added migration to add column on `plan_payments`
- Added test for new field
- Updated postman collection

FE Changes
- Added new field `paymentReconciliationFrequency` to queries
- Implemented `FrequencyForm` for new question
- Added to Readonly and CSV
- Updated unit tests



## How to test this change

- Navigate to last page of payments
- Make selections on payment reconciliation frequency question
- Verify data persists on save
- Verify render on readonly
- Verify csv export data

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
